src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -688,10 +688,39 @@
                                                 MinHeight="0"
                                                 BorderThickness="0"
                                                 ContentAlignment="Right">
-                                                <TextBlock Opacity="{x:Bind DisabledOpacity}" Style="{ThemeResource SecondaryTextStyle}">
-                                                    <Run x:Uid="PowerLauncher_AuthoredBy" />
-                                                    <Run FontWeight="SemiBold" Text="{x:Bind Author}" />
-                                                </TextBlock>
+                                                <StackPanel
+                                                    HorizontalAlignment="Right"
+                                                    Orientation="Horizontal"
+                                                    Spacing="5">
+                                                    <TextBlock Opacity="{x:Bind DisabledOpacity}" Style="{ThemeResource SecondaryTextStyle}">
+                                                        <Run x:Uid="PowerLauncher_PluginVersion" />
+                                                        <Run FontWeight="SemiBold" Text="1.0.0" />
+                                                    </TextBlock>
+                                                    <TextBlock
+                                                        Opacity="{x:Bind DisabledOpacity}"
+                                                        Style="{ThemeResource SecondaryTextStyle}"
+                                                        Text="&#x2022;" />
```

Comment:
@niels9001
Is the way of creating the separating dot good or do you prefer another way?

Reply:
I saw that there is a metadata control in the Community Toolkit. But unfortunately it doesn't allow us to change the text/link style and font. So I will go with the current solution.

Reply:
I'm retty sure we could work around the styling limitations? Or upgrading the metadatacontrol itself in the Toolkit?